editor/src/core/shared/template-path.ts

```diff
@@ -856,6 +865,40 @@ export function isFromSameSceneAs(a: TemplatePath, b: TemplatePath): boolean {
   return scenePathsEqual(scenePathPartOfTemplatePath(a), scenePathPartOfTemplatePath(b))
 }
 
-export function dynamicPathToStaticPath(path: InstancePath): StaticInstancePath {
-  return staticInstancePath(path.scene, path.element.map(extractOriginalUidFromIndexedUid))
+function dynamicScenePathToStaticScenePath(scene: ScenePath): StaticScenePath {
+  return staticScenePath(
+    scene.sceneElementPaths.map((sceneElementPath) =>
+      sceneElementPath.map(extractOriginalUidFromIndexedUid),
+    ),
+  )
+}
+
+export function dynamicPathToStaticPath(path: ScenePath): StaticScenePath
+export function dynamicPathToStaticPath(path: InstancePath): StaticInstancePath
+export function dynamicPathToStaticPath(path: TemplatePath): StaticTemplatePath
+export function dynamicPathToStaticPath(path: TemplatePath): StaticTemplatePath {
+  if (isScenePath(path)) {
+    return dynamicScenePathToStaticScenePath(path)
+  } else {
+    return staticInstancePath(
+      dynamicScenePathToStaticScenePath(path.scene),
+      path.element.map(extractOriginalUidFromIndexedUid),
+    )
+  }
+}
+
+export function scenePathContainsElementPath(scene: ScenePath, elementPath: ElementPath): boolean {
+  return scene.sceneElementPaths.some((sceneElementPath) =>
+    elementPathsEqual(sceneElementPath, elementPath),
+  )
+}
+
+export function staticScenePathContainsElementPath(
+  scene: ScenePath,
+  elementPath: ElementPath,
+): boolean {
+  return scene.sceneElementPaths.some((sceneElementPath) => {
```

Comment:
this could use `dynamicScenePathToStaticScenePath` instead of manually calling map to extractOriginalUidFromIndexedUid

Reply:
Done and I also introduced a `dynamicElementPathToStaticElementPath` since we were now doing this same map a few times.